adapters/adyoulike/adyoulike.go

```diff
@@ -0,0 +1,159 @@
+package adyoulike
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+
+	"github.com/buger/jsonparser"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+)
+
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter) (adapters.Bidder, error) {
+	return &AdYouLikeAdapter{
+		endpoint: config.Endpoint,
+	}, nil
+}
+
+type AdYouLikeAdapter struct {
```

Comment:
Please name this `adapter`. There's no need to repeat the package name and no need to export it. Please see our new guidance: https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html

Reply:
OK, will do.